docs/repo/property-pages/README.md

```diff
@@ -17,6 +17,7 @@ This documentation details the updated Project Properties UI and associated back
 To customise or extend the properties displayed for a given project start with these HOW TO guides:
 
 - [HOW TO: Add a new property page](how-to-add-a-new-property-page.md)
+- [HOW TO: Add a new Launch Profile UI](how-to-add-a-launch-profile.md)
```

Comment:
Are these "Kinds" rather than "UIs"?

Reply:
Also (minor) the `.md` file name differs from the title.

Reply:
> Are these "Kinds" rather than "UIs"?

The .xaml file only describes the UI; defining and handling the associated debug command is really the larger part of defining a new kind of Launch Profile. That's why I went with "UI" rather than "Kind" or similar, but maybe I am overthinking it.

Reply:
My concern is partly that we're talking about XAML and UI in the same document, and folks may think this is some kind of WPF XAML.

Maybe "How to define properties for a new launch profile kind".

Reply:
I ended up going with "HOW TO: Add a new Launch Profile kind".